src/backends.jl

```diff
@@ -374,6 +374,7 @@ function _initialize_backend(pkg::PlotlyBackend)
     catch
         @info "For saving to png with the Plotly backend ORCA has to be installed."
     end
+    push!(_plot_defaults, :include_mathjax => "")
```

Comment:
This should be handled differently. This line needs to be removed.

Reply:
Done